source/lib.civet

```diff
@@ -4272,71 +4270,47 @@ function wrapIIFE(expressions: StatementTuple[], async): ASTNode[] {
 }
 
 export {
```

Comment:
While we're discussing cleanup, one thing that would be nice I think is to just move `export` to the function declarations themselves. Then we will have this list in only one place, where we use it in lib.

Reply:
It's a hack to fix the type errors on `gather .type is "SomeNode"` but it causes errors when inserting strings so probably a wash.

Reply:
I don't necessarily want to export all of them, only the ones we use. Exporting them all would make dead code elimination slightly harder.